loading-button-android/src/main/java/br/com/simplepass/loading_button_lib/customViews/CircularProgressButton.java

```diff
@@ -428,6 +435,13 @@ public void startAnimation() {
             return;
         }
 
+        if (!layoutDone) {
+            shouldStartAnimation = true;
+            return;
```

Comment:
this marks the button as "animation should have been started, but is not because layout is not done